docs/handcraft.rst

```diff
@@ -1749,34 +1750,32 @@ Handcrafting the ``'TRANSFER'`` transaction
 
     condition_uri = carol_ed25519.condition.serialize_uri()
 
-    condition = {
+    output = {
         'amount': 1,
-        'cid': 0,
         'condition': {
             'details': unsigned_fulfillments_dict,
             'uri': condition_uri,
         },
-        'owners_after': (carol.public_key,),
+        'public_keys': (carol.public_key,),
     }
 
-    # The yet to be fulfilled fulfillments:
-    fulfillment = {
-        'fid': 0,
+    # The yet to be fulfilled input:
+    input_ = {
         'fulfillment': None,
-        'input': {
+        'fulfills': {
             'txid': handcrafted_car_creation_tx['id'],
-            'cid': 0,
+            'output': 0,
         },
         'owners_before': (alice.public_key, bob.public_key),
     }
 
     # Craft the payload:
     handcrafted_car_transfer_tx = {
+        'operation': 'TRANSFER',
         'asset': {'id': car_asset['id']},
         'metadata': None,
-        'operation': 'TRANSFER',
-        'conditions': (condition,),
-        'fulfillments': (fulfillment,),
+        'outputs': (output,),
+        'inputs': (input_,),
         'version': 1,
```

Comment:
version needs to be set to `0.9`

As an aside I think simply assigning the version in the `dict` is okay, that is, without the need to set a variable beforehand.